.gitignore

```diff
@@ -10,7 +10,6 @@ config.yaml
 ignored
 build
 .vscode
-package-lock.json
```

Comment:
Oh bother

Reply:
Yes! Because bazel. It needs reproducibility, and guess how they achieve it for Node libraries? Yes, by using a lockfile :( https://bazelbuild.github.io/rules_nodejs/#hermeticity-and-reproducibility